testbench/common.py

```diff
@@ -733,6 +739,14 @@ def handle_retry_test_instruction(database, request, method):
         elif items[0] == "broken-stream":
             conn = request.environ.get("gunicorn.socket", None)
             return __get_streamer_response_fn(database, method, conn, test_id)
+    broken_stream_after_bytes = (
+        testbench.common.retry_return_broken_stream_after_bytes.match(next_instruction)
+    )
+    if broken_stream_after_bytes and method == "storage.objects.get":
+        items = list(broken_stream_after_bytes.groups())
+        after_bytes = int(items[0]) * 1024
+        conn = request.environ.get("gunicorn.socket", None)
+        return __get_streamer_response_fn(database, method, conn, test_id, after_bytes)
```

Comment:
Nit: Use keyword arg rather than positional for limit.

Reply:
Good catch, thanks!